WebDriverAgentLib/Utilities/FBScreenshot.m

```diff
@@ -70,10 +71,18 @@ + (NSData *)takeWithQuality:(NSUInteger)quality
 {
   if ([self.class isNewScreenshotAPISupported]) {
     XCUIScreen *mainScreen = XCUIScreen.mainScreen;
+
+    // Note to fit to the old spec
```

Comment:
note:

![image](https://user-images.githubusercontent.com/5511591/112746571-96335d80-8fea-11eb-849f-38a2a1bfb0f4.png)

---

Could color bit differences will break existing image comparison?
(Not tested yet)

Reply:
The similarity score by comparing images feature was 1. So we can consider only the scale.